Example/Example/EditorDemoController.swift

```diff
@@ -207,9 +207,9 @@ class EditorDemoController: UIViewController {
     
     func updateTitleHeight() {
         let referenceView: UIScrollView = editingMode == .richText ? richTextView : htmlTextView
-
-        let sizeThatShouldFitTheContent = titleTextField.sizeThatFits(CGSize(width:view.frame.width - ( 2 * Constants.margin), height: CGFloat.greatestFiniteMagnitude))
+        let layoutInsets = view.layoutMargins
```

Comment:
Nitpick but could we call this `layoutMargins` instead of `layoutInsets`? Even though they are insets...